src/Couchbase.Lite.Shared/API/Sync/Replicator.cs

```diff
@@ -201,7 +201,7 @@ public void RemoveChangeListener(ListenerToken token)
         }
 
         /// <summary>
-        /// Resets the local checkpoint of the replicator, meaning that it will read all changes since the beginning
+        /// [DEPRECATED] Resets the local checkpoint of the replicator, meaning that it will read all changes since the beginning
```

Comment:
Add an `Obsolete` attribute so that using this method generates a compiler warning (make sure to indicate which method to use instead)

Reply:
Done, please see line 209.